Xamarin.Essentials/Types/PlatformExtensions/CalendarExtensions.ios.macos.cs

```diff
@@ -0,0 +1,14 @@
+﻿using System;
+using Foundation;
+
+namespace Xamarin.Essentials
+{
+    public static partial class CalendarExtensions
+    {
+        // https://developer.apple.com/documentation/foundation/nsdate
+        // NSDate minimum date is 2001/01/01
+        static DateTime iosNSDateTimeSystemZeroPoint = new DateTime(2001, 1, 1, 0, 0, 0, 0, DateTimeKind.Utc);
```

Comment:
Is there a reason to use Apple's special date as opposed to the Unix time? It also makes conversion easier. But this is just me being a bit OCD.